src/components/Form.test.js

```diff
@@ -13,191 +13,175 @@ describe('Form', () => {
     })
 
     it('submits with button click', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'Hamburg')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('Hamburg')
+        expect(startSearch).toHaveBeenCalledWith('Hamburg')
     })
 
     it('calls function with valid string', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'Hamburg')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('Hamburg')
+        expect(startSearch).toHaveBeenCalledWith('Hamburg')
     })
 
     it('calls function with valid zip code', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, '22303')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('22303')
+        expect(startSearch).toHaveBeenCalledWith('22303')
     })
 
     it('calls function with valid special characters', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'Rotenburg-Wümme')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('Rotenburg-Wümme')
+        expect(startSearch).toHaveBeenCalledWith('Rotenburg-Wümme')
     })
 
     it('doesnt call function with one or less characters', () => {
```

Comment:
doesn't / does not

Reply:
this character would end the naming string....